data/ResourceUtil.cpp

```diff
@@ -1,104 +1,106 @@
-//this file was auto-generated by res2h
+// this file was auto-generated by res2h
 
 #include "Resources.h"
 
-const size_t res2hNrOfFiles = 46;
+const uint32_t res2hNrOfFiles = 47;
```

Comment:
Regarding this change of resources, there are two questions I have:
1. Why do we need a hardcoded default image for images without an icon? To me, that seems something that a theme can (and should) define by itself.
2. Why is the type changed from `size_t` to `uint32_t`? Is that the result of a newer/different version of res2h?

As I see it now, this re-run of res2h now causes a lot of changed files (48 files out of 86 in total), but not actually a change in program behavior. Similar to style changes without a need, why are you doing this?